src/OpenTelemetry.Exporter.OpenTelemetryProtocol/.publicApi/Stable/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,6 @@
+OpenTelemetry.Exporter.OtlpExporterOptions.CertificateFile.get -> string!
```

Comment:
Shall we keep this internal and rely only on the environment variable? If there is a request, we can work on exposing these as public APIs.

Reply:
Fixed :)